apis/beacon/blocks/root.yaml

```diff
@@ -30,9 +30,8 @@ get:
                 type: object
                 properties:
                   root:
-                    allOf:
-                      - description: HashTreeRoot of BeaconBlock/BeaconBlockHeader object
-                      - $ref: '../../../beacon-node-oapi.yaml#/components/schemas/Root'
+                    $ref: '../../../beacon-node-oapi.yaml#/components/schemas/Root'
+                    description: HashTreeRoot of BeaconBlock/BeaconBlockHeader object
```

Comment:
here's a specific example of where the description is not rendered...
![Screen Shot 2022-09-01 at 8 42 09 am](https://user-images.githubusercontent.com/2967240/187797638-028e0f7b-2969-4c70-823f-726950dd8a6f.png)

basically i downloaded your PR code, started `simplehttpserver` like instructions say and changed to `dev` in "Select a definition" at the top of page, 
 - then opened `/eth/v1/beacon/blocks/{block_id}/root`, 
 - then went to 200, changed to schema tab

Expected to see the above description, it wasn't there.

This is basically exactly the problem we used to have before we used allOf, so I'm glad I could find a concrete example. I haven't gone through every single change here - there's a lot and i'm on leave :)

Hopefully that gives context about the kinds of things that can break...

Reply:
Did you bundle the oapi file before hand?

- run `swagger-cli bundle ./beacon-node-oapi.yaml -r -t yaml -o ./deploy/beacon-node-oapi.yaml`
- Serve ./deploy/beacon-node-oapi.yaml and render that in swagger

at least in my environment, `apis/beacon/blocks/root.yaml:24` prebundle is

```yaml
            type: object
            title: GetBlockRootResponse
            properties:
              execution_optimistic:
                $ref: "../../../beacon-node-oapi.yaml#/components/schemas/ExecutionOptimistic"
              data:
                type: object
                properties:
                  root:
                    $ref: '../../../beacon-node-oapi.yaml#/components/schemas/Root'
                    description: HashTreeRoot of BeaconBlock/BeaconBlockHeader object
```

after bundle 

```yaml
                type: object
                title: GetBlockRootResponse
                properties:
                  execution_optimistic:
                    type: boolean
                    example: false
                    description: 'True if the response references an unverified execution payload. Optimistic information may be invalidated at a later time. If the field is not present, assume the False value.'
                  data:
                    type: object
                    properties:
                      root:
                        description: HashTreeRoot of BeaconBlock/BeaconBlockHeader object
                        type: string
                        example: '0xcf8e0d4e9587369b2301d0790347320302cc0943d5a1884560367e8208d920f2'
                        pattern: '^0x[a-fA-F0-9]{64}$'
```

which renders the description fine

![Screenshot from 2022-09-01 13-44-25](https://user-images.githubusercontent.com/35266934/187906109-ca74ed9f-a03e-400d-ba4c-551dcb5282f3.png)

Reply:
I wasn't aware that an extra command was needed, I've been running this the way the readme says, so if that works, I'll have another look but also we should be updating the run instructions...

Reply:
That's a different screenie (is for GetStateRootResponse), I don't see the description of the root in `GetBlockHeaderResponse`...
To get mine up:
 - `npm install -g swagger-cli`
 - did your bundle command
 - ran `./simplehttpserver` and switched to `dev` version

I'm not sure if this is consistent with what you asked me to do, so wrote it all out in case I've run it incorrectly...